site/src/routes/docs/index.svelte

```diff
@@ -1,6 +1,23 @@
 <script context="module">
+	const title_replacements = {
+		'1_export_creates_a_component_prop': 'props',
```

Comment:
this seems a bit hard to discover to me compared to if it were in the markdown. though I suppose I'm slightly less worried about that because you could see something happening in the sidebar. but it might be tough to trace it back to this

Reply:
Yeah, it's not the most obvious place. However, I don't think there's a way to have it in the markdown without implementing custom parsing logic.

At least the effect of this breaking is small -- the sidebar titles use the section titles instead of the abbreviated version.

Reply:
I think this is a pretty reasonable place to have it: it is specific to this view rather than an important part of the docs.

The only other place it could go (without implementing custom parsing logic) is in the `Docs` component in site-kit, which would be very difficult to discover. One alternative could be add it to the front YAML as a map of full to short heading titles, would require a little work but doable.

I think this is good for now.